docs/configuration/arguments.md

```diff
@@ -182,7 +182,12 @@ Flags for configuring KV store based on memberlist library. This feature is expe
    Timeout for writing 'packet' data.
 - `memberlist.transport-debug`
    Log debug transport messages. Note: global log.level must be at debug level as well.
-   
+- `memberlist.gossip-to-dead-nodes-time`
+   How long to keep gossiping to the nodes that seem to be dead. After this time, dead node is removed from list of nodes.
```

Comment:
Could you also explain what happens if a dead node reappers after `gossip-to-dead-nodes-time` (ie. in the case of a network partition).